wellregistry/wellregistry/settings.py

```diff
@@ -31,12 +32,15 @@
     SECRET_KEY = os.getenv('SECRET_KEY', get_random_secret_key())
 
 # SECURITY WARNING: don't run with debug turned on in production!
-debug_setting = os.getenv('DEBUG', 'True')
+debug_setting = os.getenv('DEBUG', 'False')
 try:
     DEBUG = bool(strtobool(debug_setting))
 except ValueError:
     DEBUG = False
 
+logging.basicConfig(format="%(asctime)s [%(levelname)-8s] %(message)s",
```

Comment:
I would prefer logging be set up elsewhere. Perhaps in manage.py. It's best to try to keep this module about settings and nothing else.

Reply:
This felt like a setting but I am open.